flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/PartitionRequestClientFactory.java

```diff
@@ -120,6 +132,34 @@ public void closeOpenChannelConnections(ConnectionID connectionId) {
 		}
 	}
 
+	void connectChannelWithRetry(ConnectingChannel connectingChannel, ConnectionID connectionId)
```

Comment:
it seems this argument is never used in below logics, maybe we need to refactor the logic

Reply:
It is updated each time there is an error happens.

Reply:
Yes, actually the intellij would give this warning for never used argument.